MdePkg/Include/AArch64/ProcessorBind.h

```diff
@@ -159,9 +159,17 @@ typedef INT64 INTN;
 
 ///
 /// Page allocation granularity for AARCH64
+/// MU_CHANGE BEGIN: Enable Deprecated 4k Granularity Mode for Platforms
+/// that do not support 64k runtime allocation
 ///
 #define DEFAULT_PAGE_ALLOCATION_GRANULARITY  (0x1000)

```

Comment:
do you need this define still?

I think its covered by the #ifdef #else

Reply:
Yes, without this we won't know what size to allocate boot memory :). This is the DEFAULT vs RUNTIME granularity.